transforms/tabular/tabular-thresholding-plugin/bumpversion.cfg

```diff
@@ -0,0 +1,10 @@
+[bumpversion]
```

Comment:
Change the name of the file to `.bumpversion.cfg`.

Reply:
Renamed it